### PR TITLE
feat(explorer): add explorer script

### PR DESCRIPTION
**Pull request by DavidChild**

### CODE REVIEW

File: axon-explorer/Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+deploy: 
+	ansible-playbook start.yml
```
Comment (KaoImin) [inline axon-explorer/Makefile:2]: Missing newline at the end of the file.

File: axon-explorer/start.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  become: yes
+  become_method: sudo
+  vars_files:
+    - config.yml
+  tasks:
+  ###############################################################################################################
+    - name: star docker
+      shell: "docker run -p {{ localport }}:80 -e APP_NODE_URL={{ eth_rpc }} alethio/ethereum-lite-explorer"
+      become: yes
+      become_method: sudo
```
Comment (KaoImin) [inline axon-explorer/start.yml:13]: ditto

File: axon-explorer/config.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+eth_rpc: "http://localhost:8000"
+localport: 8190
+
```
Comment (KaoImin) [inline axon-explorer/config.yml:3]: redundant newline

**Review by KaoImin — CHANGES_REQUESTED** [review]
Please follow the conventional commits of PR title.

File: axon-explorer/start.yml (hunk 2 above)
Comment (zhengjianhui) [reply axon-explorer/start.yml:13]: Use docker-compose to manage starting and stopping, using docker directly tends to forget to clean up containers。

File: axon-explorer/Makefile (hunk 1 above)
Comment (zhengjianhui) [reply axon-explorer/Makefile:2]: At a minimum, a stop script and a data cleanup script are required。

File: axon-explorer/start.yml (hunk 2 above)
Comment (DavidChild) [reply axon-explorer/start.yml:13]: ok.

File: axon-explorer/templates/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+  ethereum-lite-explorer:
+    image: alethio/ethereum-lite-explorer
+    container_name: ethereum-lite-explorer
+    ports:
+      - "8190:80"
+    environment:
+      APP_NODE_URL: "http://localhost:8000"
```
Comment (zhengjianhui) [inline axon-explorer/templates/docker-compose.yml:10]: The docker container cannot access the axon service on the host。

Comment (zhengjianhui) [inline axon-explorer/templates/docker-compose.yml:10]: Configure isolated networks using docker's networks

**Review by zhengjianhui — CHANGES_REQUESTED** [review]

File: uni-contract-deploy/deploy.yml [hunk 5]
```diff
@@ -7,21 +7,12 @@
     - config.yml
   tasks:
   ###############################################################################################################
-    - name: Copy the axon file
-      copy:
-        src: ./templates/deploy.js
-        dest: ./deploy.js
-        mode: 0755
-    - name: Replace url
```
Comment (zhengjianhui) [inline uni-contract-deploy/deploy.yml:15]: There is a replace endpoint operation, which can be replaced in the same way as before, or can be configured to read in deploy.js

**Review by zhengjianhui — COMMENTED** [review]
Formatting Code

File: uni-contract-deploy/deploy.yml [hunk 6]
```diff
@@ -7,21 +7,12 @@
...
-      become: yes
-      become_method: sudo
     - name: yarn install
       shell: yarn install
       become: yes
       become_method: sudo
-    - name: run deploy
-      shell: node deploy.js
+    - name: deploy uniswap constracts.
+      shell: node templates/deploy.js
```
Comment (zhengjianhui) [inline uni-contract-deploy/deploy.yml:15]: Move to the root directory

File: uni-contract-deploy/deploy.js [hunk 7]
```diff
@@ -3,15 +3,18 @@ const ethers = require("ethers");
...
-
-const endpoint = 'http://127.0.0.1:8000';
-const hexPrivateKey = '0x37aa0f893d05914a4def0460c0a984d3611546cfb26924d7a7ca6e0db9950a2d';
+const WETH9 = require('.././uni-test/artifacts/contracts/Weth.sol/WETH9.json');
+const UniswapV2Factory = require('.././uni-test/artifacts/contracts/Factory.sol/UniswapV2Factory.json');
+const UniswapV2Router02 = require('.././uni-test/artifacts/contracts/UniswapV2Router02.sol/UniswapV2Router02.json');
+const UniswapV2Pair = require('.././uni-test/artifacts/contracts/Factory.sol/IUniswapV2Pair.json');
+const Multicall = require('.././uni-test/artifacts/contracts/MultilCall.sol/Multicall.json');
+const configdata = fs.readFileSync('config.yml','utf8');
+const configsetting=yaml.load(configdata);
```
Comment (zhengjianhui) [inline uni-contract-deploy/deploy.js:12]: Formatting the js code

File: uni-contract-deploy/deploy.js [hunk 8]
```diff
@@ -3,15 +3,18 @@ const ethers = require("ethers");
...
+console.log(configsetting);
+console.log(configsetting.node_address);
+console.log(configsetting.hex_private_key);
+const endpoint = configsetting.node_address;
+const hexPrivateKey =configsetting.hex_private_key;
```
Comment (zhengjianhui) [inline uni-contract-deploy/deploy.js:17]: Formatting the js code

**Review by zhengjianhui — APPROVED** [review]

**Review by KaoImin — APPROVED** [review]

**Pull request merged**